ace-manifest.json

```diff
@@ -48,6 +48,219 @@
       "args": [],
       "aliases": [],
       "flags": []
+    },
+    "db:seed": {
```

Comment:
Interessting, got any docs for this manifest thing?

Reply:
https://docs.adonisjs.com/guides/ace-commandline#generating-the-ace-manifest-file